btp-core/src/main/java/org/interledger/btp/BtpError.java

```diff
@@ -20,10 +20,9 @@
  * =========================LICENSE_END==================================
  */
 
-import org.interledger.core.Immutable;
-
 import org.immutables.value.Value.Default;
 import org.immutables.value.Value.Derived;
+import org.interledger.core.Immutable;
```

Comment:
Do these re-arrangements pass checkstyle for you? I still think our checkstyle is slightly broken, but if you run Checkstyle from Intellij you can see there's technically an ordering that these imports need to be in (I need to research which way is correct).

Reply:
`IntelliJ` kept complaining about the previous ordering and wanted to use this ordering which in fact seems to be better because `org.im..` < `org.in..`

Reply:
@sudheesh001 have you imported the `checkstyle.xml` file from the project directory? When I do that and run checkstyle in Intellij, I get this error:

`Import statement for 'org.interledger.core.Immutable' is in the wrong order. Should be in the 'SPECIAL_IMPORTS' group, expecting group 'STANDARD_JAVA_PACKAGE' on this line. (25:0) [CustomImportOrder]`

Checkstyle only works when the ordering is like this (which is what's in `master`):

```
import org.interledger.core.Immutable;

import org.immutables.value.Value.Default;
import org.immutables.value.Value.Derived;

import java.time.Instant;
```

I also get these same results when running in `mvn clean install` on the command-line.